autoPyTorch/pipeline/components/training/trainer/base_trainer_choice.py

```diff
@@ -211,8 +280,15 @@ def fit(self, X: Dict[str, Any], y: Any = None, **kwargs: Any) -> autoPyTorchCom
             y=y,
             **kwargs
         )
+        # Add snapshots to base network to enable
+        # predicting with snapshot ensemble
+        self.choice: autoPyTorchComponent = cast(autoPyTorchComponent, self.choice)
+        if self.choice.use_snapshot_ensemble:
+            X['network_snapshots'].extend(self.choice.model_snapshots)
 
-        return cast(autoPyTorchComponent, self.choice)
+        # TODO: when have the optimizer code, the pynisher object might have failed
```

Comment:
I think we can safely remove this comment

Reply:
Thanks for finding it, we should remove it.